tests/redirects/map_globalconf.py

```diff
@@ -187,7 +187,7 @@
         url_test("/{m,{firefox/,}mobile}/faq/", "https://support.mozilla.org/products/mobile"),
         # bug 885799, 952429
         url_test("/projects/calendar/holidays.html", "https://www.thunderbird.net/calendar/holidays/"),
-        url_test("/en-US/projects/calendar/random/stuff/", "https://www.thunderbird.net/calendar/"),
+        url_test("/projects/calendar/random/stuff/", "https://www.thunderbird.net/calendar/"),
```

Comment:
**note:** This is the only test I had to hack to make it fit. I think it's OK, particularly given the lower proportion of visits it _probably_ gets